frb_dart/lib/flutter_rust_bridge.dart

```diff
@@ -1,3 +1,11 @@
+import 'package:flutter_rust_bridge/src/basic.dart';
+
 export 'src/flutter_rust_bridge_io.dart'
     if (dart.library.html) 'flutter_rust_bridge_web.dart';
 export 'src/platform_independent.dart';
+
+class Anyhow implements FrbException {
```

Comment:
👀  is this used somewhere

(If yes we may need to move it to somewhere else instead of this file)